gu-net/src/types.rs

```diff
@@ -122,8 +125,7 @@ impl str::FromStr for NodeId {
 }
 
 static HEX_CHARS: [u8; 16] = [
-    '0' as u8, '1' as u8, '2' as u8, '3' as u8, '4' as u8, '5' as u8, '6' as u8, '7' as u8,
-    '8' as u8, '9' as u8, 'a' as u8, 'b' as u8, 'c' as u8, 'd' as u8, 'e' as u8, 'f' as u8,
+    b'0', b'1', b'2', b'3', b'4', b'5', b'6', b'7', b'8', b'9', b'a', b'b', b'c', b'd', b'e', b'f',
```

Comment:
noice 👍 
https://doc.rust-lang.org/reference/tokens.html#byte-literals
didn't know that